### PR TITLE
Test proxying + invoking Grackle demo

### PR DESCRIPTION
**Pull request by rpiaggio**
No need to merge, just testing things out.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,35 +1,35 @@
 {
-    "buildpacks": [
+  "buildpacks": [
```
Comment (cquiroz) [inline app.json]: This probably took a long time to get right 👍

File: common/src/main/scala/explore/model/RootModel.scala [hunk 2]
```diff
@@ -14,6 +14,7 @@ import monocle.macros.Lenses
 import cats.effect.concurrent.Ref
 import japgolly.scalajs.react._
 import diode.data._
+import explore.util.Pot._
```
Comment (cquiroz) [inline common/src/main/scala/explore/model/RootModel.scala]: Did you internalize `Pot`?

File: common/src/main/webpack/dev.webpack.config.js [hunk 3]
```diff
@@ -37,7 +37,19 @@ const Web = Merge(
       host: "0.0.0.0",
       hot: true,
       contentBase: [__dirname, parts.rootDir],
-      historyApiFallback: true
+      historyApiFallback: true,
+      proxy: {
+        "/api/tasks": {
```
Comment (cquiroz) [inline common/src/main/webpack/dev.webpack.config.js]: nice 👍

File: static.json [hunk 4]
```diff
@@ -3,5 +3,13 @@
     "clean_urls": false,
     "routes": {
       "/**": "index.html"
+    },
+    "proxies": {
+      "/api/tasks": {
+        "origin": "https://todo-mongo-graphql-server.herokuapp.com"
+      },
+      "/api/grackle-demo": {
+        "origin": "https://${GRACKLE_DEMO_BASE}"
```
Comment (cquiroz) [inline static.json]: Does this come from heroku?

Comment (rpiaggio) [reply static.json]: We define it as an environment variable in Heroku. This allows us to use the same app pointing to different servers in staging and production.

File: common/src/main/scala/explore/model/RootModel.scala (hunk 2 above)
Comment (rpiaggio) [reply common/src/main/scala/explore/model/RootModel.scala]: Not yet. For the moment in that object we have some utils for working with it.

**Comment by rpiaggio** [comment]
I'll go ahead and merge so we can reuse the mechanism.

**Pull request merged**